core/peer/config.go

```diff
@@ -114,6 +114,15 @@ type Config struct {
 	// qscc system chaincode requests.
 	LimitsConcurrencyQSCC int
 
+	// LimitsConcurrencyEndorserService sets the limits for concurrent requests sent to
+	// endorser service that handles chaincode deployment, query and invocation,
+	// including both user chaincodes and system chaincodes.
+	LimitsConcurrencyEndorserService int
```

Comment:
Nit.
The following view might be subjective. We can improve the readability of `LimitConcurrencyEndorserService` and `LimitConcurrencyDeliveryService` variables. The following would have been better to read IMO (as concurrency limit or rate limit is widely used terms)
```suggestion
	EndorserServiceConcurrencyLimit int
```
I understand that the name is chosen to match the one in the `core.yaml` (limits.concurrency.endorserService). To maintain consistency, we can leave it as is unless we decide to change the newly introduced config parameters in `core.yaml`.

Reply:
To maintain consistency, leave it as is unless the config params are changed in core.yaml.